package.json

```diff
@@ -67,17 +68,26 @@
     "css-loader": "^3.4.0",
     "cssnano": "^4.1.10",
     "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
```

Comment:
@nicholasio Question: why all `eslint-*` related packages needs to be in scaffold? Why not in the eslint package itself?

Reply:
It's the official eslint recommendation for creating shareable configs, I actually think eslint isn't able to resolve the deps if they weren't installed by the application. But I'll double check.

Reply:
@samikeijonen Confirmed that plugin dependencies on a shareable configs needs to be a peer dps: https://eslint.org/docs/developer-guide/shareable-configs